apps/web/src/views/Notifications/hooks/sendPushNotification.ts

```diff
@@ -43,17 +43,20 @@ const useSendPushNotification = (): IUseSendNotification => {
 
         const existingSubscription = await registration.pushManager.getSubscription()
 
+        if (existingSubscription) {
+          // Unsubscribe the user from the existing subscription // fixes brokeb subscriptuon
```

Comment:
mind the typo in the comments

Reply:
why exactly do we need to resubscribe?

Reply:
no this is a service thats not anything to do with walletconnect. 

this is for webpush browser alerts so users get notified if they recieve a pancakeswap notification from being in other tabs outside of PCS. basically the backend uses this subscription to send webpush alerts from the notification hub api. but i updated the vapid keys. a user can only have one subscription at a time. so this unsubscribe check will unsubscribe the user if they have an existing subscription and resubscribe them. 

this is done to prevent an error for incorrect auth key from happening in the backend. we dont need to push this now we can do it monday